src/components/station/WisStation.vue

```diff
@@ -48,6 +48,10 @@ export default defineComponent({
   },
   methods: {
     onReady() {
+      if (!this.features.features || !this.features.features.length) {
+        return
+      }
```

Comment:
This fixes the error where it says "bad sort property". That error was happening due to us trying to sort a collection with no features